bonsai/BudgetTransactions/BudgetTransactions.swift

```diff
@@ -0,0 +1,127 @@
+//
+//  BudgetTransactions.swift
+//  bonsai
+//
+//  Created by hoang on 08.12.2021.
+//
+
+import SwiftUI
+
+struct VisualEffectView: UIViewRepresentable {
+    var effect: UIVisualEffect?
+    func makeUIView(context: UIViewRepresentableContext<Self>) -> UIVisualEffectView { UIVisualEffectView() }
+    func updateUIView(_ uiView: UIVisualEffectView, context: UIViewRepresentableContext<Self>) { uiView.effect = effect }
+}
+
+struct TransactionData: Identifiable {
+    let id = UUID()
+    let dateTitle: String
+    let items: [TransactionItem]
+}
+
+struct TransactionItem: Identifiable, Hashable {
+    let id = UUID()
+    let image: UIImage
+    let name: String
+    let value: Double
+}
+
+struct RoundedCorner: Shape {
+
+    var radius: CGFloat = .infinity
```

Comment:
I've created the same thing in https://github.com/appbonsai/bonsai./pull/13
When merging please take care to remove one of the implementations as they duplicating each other

Reply:
will be deleted after merge BS-9 or before :)

Reply:
👍

Reply:
removed duplicates